draft-ietf-rats-architecture.md

```diff
@@ -697,6 +697,13 @@ for by hardware or by ROM code, especially if such hardware is
 physically resistant to hardware tampering.  The component that is
 implicitly trusted is often referred to as a Root of Trust.
 
+A conveyance protocol that is authenticated and integrity protected can be used
```

Comment:
I assume what is meant here is:

* evidence is protected
* remote peers are (mutually) authenticated